fab3/filters.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package fab3
+
+import (
+	"encoding/hex"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/hyperledger/fabric-chaincode-evm/fab3/types"
+)
+
+type filterEntry interface {
+	LastAccessTime() time.Time
+	Filter(s *ethService) ([]interface{}, error)
+}
+
+type logsFilter struct {
+	logArgs         *types.GetLogsArgs
+	latestBlockSeen uint64
+	lastAccessTime  time.Time
+}
+
+func (f *logsFilter) LastAccessTime() time.Time {
+	return f.lastAccessTime
+}
+
+func (f *logsFilter) Filter(s *ethService) ([]interface{}, error) {
+	s.logger.Debug("lastblockseen before filtering", f.latestBlockSeen)
```

Comment:
I might have missed this, where is `f.latestBlockSeen` updated?

Reply:
resolved above, I assume?